.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+blank_issues_enabled: true
+contact_links:
+- name: Ask a question (GitHub Discussions)
+  url: https://github.com/lima-vm/lima/discussions
+- name: Chat with Lima users and developers
+  url: https://slack.rancher.io/
+  about: Rancher Users slack has `#lima` channel, colocated with `#rancher-desktop` channel (Lima is not Rancher's product)
```

Comment:
I would prefer: "(Lima is not a Rancher project)".

Technically it should also be "SUSE" as Rancher does not exist as a separate identity anymore, but I think it is fine here because it is in the "Rancher Users Slack" name. But Rancher/SUSE is providing professional services (support, consulting); all the software is in FOSS projects.

So this should just be pointing out that Lima is not a Rancher project.

Well, I think. Now it feels like nit-picking. Change or apply as-is! 😄

Reply:
fixed